docs/features/service.md

```diff
@@ -24,68 +24,68 @@ The following parameters can be modified by governance(governance.md)
 
 # disable a available service binding
-iriscli service disable --chain-id=<chain-id>  --from=x --fee=0.3iris --def-chain-id=<chain-id> --service-name=test-service
+iriscli service disable --chain-id=<chain-id>  --from=<key_name> --fee=0.3iris --def-chain-id=<def-chain-id> --service-name=<service_name>
 
 # enable an unavailable service binding
-iriscli service enable --chain-id=<chain-id>  --from=x --fee=0.3iris --def-chain-id=<chain-id> --service-name=test-service --deposit=1iris
+iriscli service enable --chain-id=<chain-id>  --from=<key_name> --fee=0.3iris --def-chain-id=<def-chain-id> --service-name=<service_name> --deposit=100iris
 
 # refund all deposit from a service binding
-iriscli service refund-deposit --chain-id=<chain-id>  --from=x --fee=0.3iris --def-chain-id=<chain-id> --service-name=test-service
+iriscli service refund-deposit --chain-id=<chain-id>  --from=<key_name> --fee=0.3iris --def-chain-id=<def-chain-id> --service-name=<service_name>
 ```
 
 ### Service Invocation
 
-If the service consumer needs to initiate a service invocation request, the service fee specified by the service provider needs to be paid. The service provider needs to respond to the service request within the block height defined by `MaxRequestTimeout`. If the service provider does not respond in time, the deposit of the 'SlashFraction' ratio will be deducted from the service provider's service binding deposit. If the service call is responded normally, the system will deduct the system tax of the `ServiceFeeTax` ratio from the service fee called by the service, and add the remaining service fee to the service provider's incoming pool. If the service call does not respond in time, the service fee for the service call will be refunded to the service consumer's return pool. The service provider/consumer can initiate the `withdraw-fees`/`refund-fees` transaction to retrieve all of the tokens in the incoming/return pool.
+If the service consumer needs to initiate a service invocation request, the service fee specified by the service provider needs to be paid. The service provider needs to respond to the service request within the block height defined by `MaxRequestTimeout`. If the service provider does not respond in time, the deposit of the 'SlashFraction' ratio will be deducted from the service provider's service binding deposit. If the service call is responded normally, the system will deduct the `ServiceFeeTax` ratio from the service fee, and add the remaining service fee to the service provider's incoming pool. If the service call does not respond in time, the service fee will be refunded to the service consumer's return pool. The service provider/consumer can initiate the `withdraw-fees`/`refund-fees` transaction to retrieve all of the tokens in the incoming/return pool.
```

Comment:
If the service provider does not respond in time, the deposit of the 'SlashFraction' ratio will be deducted from the service provider's service binding deposit, and the service fee will be refunded to the service consumer's return pool.

Reply:
fixed